Curl depdencies (may be) missing for some with `Develop without docker` section

FYI on Ubuntu 20.04 (fairly fresh install) following [Dev w/o Docker](https://github.com/rapidsai/node-rapids/blob/main/docs/setup.md#develop-without-docker) I run into this error:

`Could NOT find CURL (missing: CURL_LIBRARY CURL_INCLUDE_DIR) 
CMake Error at CMakeLists.txt:477 (message):
  CMAKE_USE_SYSTEM_CURL is ON but a curl is not found!
`

Seems base CURL install is missing some dependencies - I resolved it with:
`sudo apt install libssl-dev libcurl4-openssl-dev`

--- a/modules/core/bin/install-deps/debian.sh
+++ b/modules/core/bin/install-deps/debian.sh
@@ -93,9 +93,9 @@ if [ -n "$INSTALL_CMAKE" ]; then
             'APT_DEPS=${APT_DEPS:+$APT_DEPS }qt5-default; CMAKE_OPTS=${CMAKE_OPTS:+$CMAKE_OPTS }--qt-gui;';
     fi
 
-    if [ "$(check_apt libcurl4-openssl-dev)" -eq "0" ]; then
-        ask_before_install "Missing libcurl4-openssl-dev needed by CMake. Install it now (y/n)?" \
-            'APT_DEPS=${APT_DEPS:+$APT_DEPS }libcurl4-openssl-dev; CMAKE_OPTS=${CMAKE_OPTS:+$CMAKE_OPTS }--system-curl;';
+    if [ "$(check_apt libcurl4-openssl-dev)" -eq "0" ] || [ "$(check_apt libssl-dev)" -eq "0" ]; then
+        ask_before_install "Missing libcurl4-openssl-dev and/or libssl-dev needed by CMake. Install them now (y/n)?" \
+            'APT_DEPS=${APT_DEPS:+$APT_DEPS }libcurl4-openssl-dev libssl-dev; CMAKE_OPTS=${CMAKE_OPTS:+$CMAKE_OPTS }--system-curl;';
     fi
 fi
 

